compiler/ballerina-parser/src/test/resources/expressions/byte-array-literal/base64_literal_assert_05.json

```diff
@@ -0,0 +1,82 @@
+{
+    "kind": "BYTE_ARRAY_LITERAL",
+    "children": [
+        {
+            "kind": "BASE64_KEYWORD"
+        },
+        {
+            "kind": "BACKTICK_TOKEN"
+        },
+        {
+            "kind": "LIST",
+            "children": [
+                {
+                    "kind": "BASE64_GROUP",
+                    "children": [
+                        {
+                            "kind": "BASE64_CHAR",
+                            "value": "A"
+                        },
+                        {
+                            "kind": "BASE64_CHAR",
+                            "value": "w"
+                        },
+                        {
+                            "kind": "BASE64_CHAR",
+                            "value": "s"
+                        },
+                        {
+                            "kind": "BASE64_CHAR",
+                            "value": "z"
+                        }
+                    ]
+                },
+                {
+                    "kind": "BASE64_GROUP",
+                    "children": [
+                        {
+                            "kind": "BASE64_CHAR",
+                            "value": "l"
+                        },
+                        {
+                            "kind": "BASE64_CHAR",
+                            "value": "q"
+                        },
+                        {
+                            "kind": "BASE64_CHAR",
+                            "value": "2"
+                        },
+                        {
+                            "kind": "BASE64_CHAR",
+                            "value": "r"
+                        }
+                    ]
+                },
+                {
+                    "kind": "PADDED_BASE64_GROUP",
+                    "children": [
+                        {
+                            "kind": "BASE64_CHAR",
+                            "value": "t"
+                        },
+                        {
+                            "kind": "BASE64_CHAR",
+                            "value": "y"
+                        },
+                        {
+                            "kind": "BASE64_CHAR",
+                            "value": "s"
+                        },
+                        {
+                            "kind": "PADDING_CHAR",
+                            "value": "\u003d"
+                        }
```

Comment:
`=` is encoded like this when converting to JSON string. I think that's okay. 
Or 
since `PADDING_CHAR` is always `=`, we can  use `getSyntaxToken(SyntaxKind.PADDING_CHAR)` instead of `getLiteral(SyntaxKind.PADDING_CHAR)` in the lexer thus the `value` property will not be here.